pkg/readiness/object_tracker.go

```diff
@@ -300,7 +305,7 @@ func (t *objectTracker) Satisfied() bool {
 	if !needMutate {
 		// Read lock to prevent concurrent read/write while logging readiness state.
 		t.mu.RLock()
-		log.V(1).Info("readiness state", "gvk", t.gvk, "satisfied", fmt.Sprintf("%d/%d", len(t.satisfied), len(t.expect)+len(t.satisfied)))
+		log.V(1).Info("readiness state unsatisfied", "gvk", t.gvk, "satisfied", fmt.Sprintf("%d/%d", len(t.satisfied), len(t.expect)+len(t.satisfied)))
```

Comment:
what would be the concern here?

I figured the addition of the "unsatisfied" is helpful to not "break" any log indexers.

Also, in debugging, this line was really confusing bc it would say `satisfied 1/1` but we return false in this block bc of, in my case, `t.populated` was false.

Is there another way for me to make this log line more useful if we cannot modify the message? maybe adding the `t.populated`?

Reply:
I don't think the extra verbiage is accurate, since this is meant to be a dump of the readiness state. Adding populated SGTM